platform/api.search/src/org/netbeans/modules/search/ui/BasicSearchResultsPanel.java

```diff
@@ -18,15 +18,26 @@
  */
 package org.netbeans.modules.search.ui;
 
+import java.beans.PropertyChangeEvent;
+import javax.swing.BoxLayout;
+import javax.swing.JPanel;
+import javax.swing.JSplitPane;
 import org.netbeans.modules.search.BasicComposition;
+import org.netbeans.modules.search.ContextView;
+import org.netbeans.modules.search.FindDialogMemory;
 import org.netbeans.modules.search.ResultModel;
 import org.openide.nodes.Node;
+import org.openide.util.RequestProcessor;
 
 /**
  *
  * @author jhavlin
  */
 public class BasicSearchResultsPanel extends BasicAbstractResultsPanel {
+    private static final RequestProcessor RP =
+            new RequestProcessor(BasicSearchResultsPanel.class.getName());
+    private final RequestProcessor.Task SAVE_TASK = RP.create(new BasicSearchResultsPanel.SaveTask());
```

Comment:
this could probably use the `RequestProcessor.getDefault().create..` most likely, since updating a single attribute should be super fast and doesn't really happen that often.

Reply:
Done.